cerbero/packages/disttarball.py

```diff
@@ -121,41 +141,101 @@ def _create_tarball_stripped(self, output_dir, package_type, files, force,
         prefix_restore = self.prefix
         self.prefix = tmpdir
         tarball = self._create_tarball(output_dir, package_type,
-                                       files, force, package_prefix)
+                                       files, force, package_prefix,
+                                       relocatable, lib64_link)
         self.prefix = prefix_restore
         shutil.rmtree(tmpdir)
 
         return tarball
 
     def _create_tarball(self, output_dir, package_type, files, force,
-                        package_prefix):
-        filename = os.path.join(output_dir, self._get_name(package_type))
+                        package_prefix, relocatable, lib64_link):
+        filename = os.path.join(output_dir, self.get_name(package_type))
         if os.path.exists(filename):
             if force:
                 os.remove(filename)
             else:
                 raise UsageError("File %s already exists" % filename)
+
+        tar_files = []
+        restore_files = []
+        inodes_copied = []
+
+        for f in files:
+            filepath = os.path.join(self.prefix, f)
+            stat = os.stat(filepath)
+            if relocatable and not os.path.islink(filepath):
+                if is_text_file(filepath) and stat.st_ino not in inodes_copied:
+                    if stat.st_nlink > 1:
+                        inodes_copied.append(stat.st_ino)
+                    shutil.copy(filepath, filepath + RESTORE_SUFFIX)
+                    restore_files.append(filepath)
+                    with open(filepath, 'rb+') as fo:
+                        content = fo.read()
+                        content = replace_prefix_in_bytes(self.config.prefix, content, 'CERBERO_PREFIX')
+                        fo.seek(0)
+                        fo.write(content)
+                        fo.truncate()
+                        fo.flush()
+                        tar_files.append(filepath)
+                else:
+                    tar_files.append(filepath)
+            else:
+                tar_files.append(filepath)
+
+        # This link allows to rpm packager to install the libs
```

Comment:
Is this really needed? For RPM target distros lib_suffix = 64 should be used in the .cbc
Otherwise a lib32 link should also be needed

Reply:
I have no idea, to be honest. That specific chunk is taken directly from https://github.com/fluendo/cerbero/blame/master/cerbero/packages/distarchive.py#L142-L153 and was added by Stephane here https://github.com/fluendo/cerbero/commit/d25a81422411192309a8def5b363e586a0e43fc8